features/facebookImport/src/model/entities/message-threads-group.js

```diff
@@ -2,19 +2,19 @@ import MessageThread from "./message-thread.js";
 
 export default class MessageThreadsGroup {
     constructor() {
-        this._messagesThreads = [];
+        this._messagesThreads = {};
     }
 
     get messagesThreads() {
-        return this._messagesThreads;
+        return Object.values(this._messagesThreads);
     }
 
     get messageThreadsCount() {
-        return this._messagesThreads.length;
+        return Object.keys(this._messagesThreads).length;
```

Comment:
Well, this is tricky. Keys and values of this._messageThreads should have the same length. So wouldn't this be equivalent to 

    this.messageThreads().length

?

Also, do you need a function for that?

Reply:
Yep it would be equivalent, will change it to make it look neater :). do you mean if we need a getter for the messageThreadsCount ?

Reply:
Actually the other way round, just get rid of this function. But a getter might be a nice alternative.